utils/helpers.py

```diff
@@ -158,7 +158,7 @@ def get_assets(
             "labels": [
                 line
                 for line in sorted(a["labels"], key=lambda l: l["createdAt"])
-                if line["labelType"] in label_types
+                if (line["labelType"] in label_types and line["jsonResponse"])
```

Comment:
Added this check since if the last label was skipped, it would return an asset with no jsonResponse